controller/api/destination/watcher/endpoints_watcher.go

```diff
@@ -681,3 +923,43 @@ func diffAddresses(oldAddresses, newAddresses AddressSet) (add, remove AddressSe
 	}
 	return
 }
+
+func endpointSliceAccess(k8sClient kubernetes.Interface) error {
```

Comment:
does it actually make sense to retru nan error from here. From what I see you just want to know whether you can use slices or not, right? I think a bool should suffice there WDYT ?